perf.py

```diff
@@ -217,13 +228,19 @@ def get_target_ip(self) -> str:
             external_pod_ip = self.get_podman_ip(self.server.pod_name)
             logger.debug(f"get_target_ip() External connection to {external_pod_ip}")
             return external_pod_ip
+        elif self.connection_mode in (ConnectionMode.MULTI_NETWORK, ConnectionMode.MULTI_HOME):
```

Comment:
there are various lines in the PR that would be reformatter by `python-black`. Please run python-black on it.

Also note the github action, which checks for this style. Basically, fix issues in the "Check style, test / build" github test.

Reply:
Reformatted using python-black